src/main/kotlin/com/kotlindiscord/database/TableDefinitions.kt

```diff
@@ -0,0 +1,68 @@
+package com.kotlindiscord.database
+
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.dao.id.LongIdTable
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.`java-time`.datetime
+import org.postgresql.util.PGobject
+
+@Suppress("MagicNumber")
+object Roles : IdTable<Long>() {
+    override val id = long("id").entityId()
+    val name = varchar("name", 100)
+    val colour = integer("colour")
+    override val primaryKey = PrimaryKey(id, name = "PK_Role_ID")
+}
+
+object UserRoles : Table() {
+    private val user = reference("user", Users)
```

Comment:
Property order/access modifiers probably need thinking about. Perhaps put overrides at the top and separate them visually from the rest with a blank line?

Also, did you intend for these to be private?

Reply:
This shouldn't have been private, that's fixed.

As for override vals, I've now moved all of the overrides to the bottom, as some overrides rely on other properties already being defined; that's the way the Exposed docs do it for Primary Keys.